src-vue/src/components/LanguageSelector.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <el-select v-model="value" class="m-2" 
+        :placeholder="$t('settings.language_select')" size="large"
+        @change="onChange"
+    >
+        <el-option
+            v-for="item in options"
+            :key="item.value"
+            :label="item.label"
+            :value="item.value"
+        />
+  </el-select>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { Store } from 'tauri-plugin-store-api';
+const persistentStore = new Store('flight-core-settings.json');
+
+export default defineComponent({
+    name: 'LanguageSelector',
+    data: () => ({
+        value: '',
+        options: [
+            {
+                value: 'en',
```

Comment:
Shouldn't it technically be `fr-FR` and `en-UK` (I think I put most stuff in UK spelling) to allow for variations of a language, e.g. UK vs US or Spanish vs Mexican Spanish?

Reply:
So turns out the one I meant is [RFC 5646](https://www.rfc-editor.org/rfc/rfc5646.html). Anyway, ISO-3166 is probably fine for now ^^